### ISSUE TITLE
[BUG] TDRC tag not captured as "year"

### ISSUE THREAD

**Issue by RanTalbott** [devsnd/tinytag#162]
**Describe the bug**
I have a program that captures album information from my collection to a spreadsheet. Files tagged with TDRC came up with an empty year cell.

This change to 1.8.1 caused tinytag to capture the tag to .year, instead of "extra.tdrc"
490c490
<         'TYER': 'year', 'TYE': 'year',
---
>         'TYER': 'year', 'TYE': 'year', 'TDRC': 'year',
501a502
> ##        'TDRC': 'extra.tdrc',

**Comment by RanTalbott** [comment]
Here's the diff without the editor's garbling

```
490c490
<         'TYER': 'year', 'TYE': 'year',
---
>         'TYER': 'year', 'TYE': 'year', 'TDRC': 'year',
501a502
> ##        'TDRC': 'extra.tdrc',
```

**Comment by mathiascode** [comment]
Thanks for the report. This was previously reported in https://github.com/devsnd/tinytag/issues/155, and is fixed in master (248b2eba7b0664b8b6e35257e216079865e4996f).

Hopefully a new release will be available in the near future, when @devsnd has time.
